helm/auth-bundle/templates/_helpers.tpl

```diff
@@ -45,3 +45,59 @@ giantswarm.io/service-type: managed
 application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
+
+{{- define "default.config.oidc.connectors" -}}
+  connectors:
+  {{- range $connector := .connectors }}
+  - id: {{ $connector.id }}
+    connectorName: {{ $connector.connectorName }}
+    connectorType: {{ $connector.connectorType }}
+    connectorConfig: |-
+    {{- (merge (fromYaml $connector.connectorConfig) (dict "redirectURI" (printf "https://dex.%s/callback" $.baseDomain))) | toYaml | nindent 6 }}
+  {{- end }}
+{{- end -}}
+
+{{- define "default.config" -}}
+{{- if .Values.defaultConfig -}}
+dex-app:
+  userConfig:
+    configMap:
+      values: |
+        isWorkloadCluster: {{ ne .Values.managementCluster .Values.clusterID }}
+        deployDexK8SAuthenticator: {{ eq .Values.defaultConfig.deployDexK8SAuthenticator true }}
+        {{ if .Values.defaultConfig.oidc.expiry -}}
+        oidc:
+          expiry:
+            {{- .Values.defaultConfig.oidc.expiry | toYaml |  nindent 12 -}}
+        {{- end }}
+    secret:
+      values: |
+        oidc:
+{{- if .Values.defaultConfig.oidc.customer }}
+          customer:
+            {{- (include "default.config.oidc.connectors" (dict "connectors" .Values.defaultConfig.oidc.customer.connectors "baseDomain" .Values.baseDomain)) | nindent 12 }}
+{{ end -}}
+{{- if .Values.defaultConfig.oidc.giantswarm }}
```

Comment:
Are giantswarm connectors generally defined in the default config?
I am asking since if not it might make sense to omit them to make this configuration more simple for the customer

Reply:
It might make sense to omit them completely. I wasn't sure whether it's safe, so I kept them as optional.

Reply:
hmmm do you have a specific concern regarding safety?

Reply:
I don't have any specific concern or use case in mind. I just think that in some cases somebody might want to define Giant Swarm connectors in the default config (e.g. when creating a workload cluster to test something), and if the auth bundle throws an error in that case, the user might find it confusing.

Reply:
That's fair. I'm alright with either